content/developer/howtos/localization.rst

```diff
@@ -0,0 +1,374 @@
+            "data/account_fiscal_position_template_data.xml",
+            "data/account_account_template_post_data.xml",
+
+            "data/account_chart_post_data.xml",
+            "data/account_chart_template_try_loading.xml",
+
+            # Views and others
+            "views/xxxmodel_views.xml"
+        ],
+        "demo": [
+            "demo/demo_company.xml",
+        ]
+    }
+
+
+In the first file `data/account_chart_template_data.xml` we give name for the Chart of Accounts and set some basic fields.
+
+.. seealso::
+   :ref:`Chart Template References <reference/account_chart_template>`
+
+.. example::
+  `addons/l10n_ch/data/l10n_ch_chart_data.xml <https://github.com/odoo/odoo/blob/15.0/addons/l10n_ch/data/l10n_ch_chart_data.xml>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_ch/data/l10n_ch_chart_data.xml
+    :language: xml
+    :start-at: l10nch_chart_template
+    :end-at: </record>
+
+
+.. note::
+
+  Recommended **xmlid** for the record is `chart_template`.
+  If you need many chart of accounts, you can add some suffixes, i.e. `chart_template_XXX`.
+
+
+Chart of Accounts
+=================
+
+Account tags
+------------
+
+.. seealso::
+   :ref:`Account Tag References <reference/account_account_tag>`
+
+Tags are a way to sort accounts.
+For example, imagine you want to create a financial report having multiple lines but you have no way to find a rule to dispatch the accounts according their ``code`` or ``name``.
+The solution is the usage of tags, one for each report line, to filter accounts like you want.
+
+Put the tags in `data/account_account_tag_data.xml` file.
+
+.. example::
+  `addons/l10n_lt/data/account.account.template.csv <https://github.com/odoo/odoo/blob/15.0/addons/l10n_lt/data/account.account.template.csv>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_lt/data/account.account.template.csv
+    :language: csv
+    :end-at: account_account_template_1201  
+
+.. example::
+  `addons/l10n_at/data/account_account_template.xml <https://github.com/odoo/odoo/blob/15.0/addons/l10n_at/data/account_account_template.xml>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_at/data/account_account_template.xml
+    :language: xml
+    :start-at: chart_at_template_0010
+    :end-at: </record>
+
+Accounts
+--------
+
+.. seealso::
+   - :ref:`Account References <reference/account_account>`
+   - :doc:`/applications/finance/accounting/payables`
+   - :doc:`/applications/finance/accounting/receivables`
+
+Obviously, *Chart of Accounts* cannot exist without *Accounts*. You need to specify them in `data/account.account.template.csv`.
+
+.. example::
+  `addons/l10n_ch/data/account.account.template.csv <https://github.com/odoo/odoo/blob/15.0/addons/l10n_ch/data/account.account.template.csv>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_ch/data/account.account.template.csv
+    :language: csv
+    :end-at: ch_coa_1171
+
+CSV is prefered but you may use XML format instead.
```

Comment:
a little note on the perf diff?

Reply:
There is none (tested one or two years ago, after 13.0). 
It is just for readability.

Reply:
IIRC, csv records are created in batch, which isn't the case for xml records :thinking: